mintapi/api.py

```diff
@@ -1744,29 +1748,48 @@ def initiate_account_refresh(email, password):
     return mint.initiate_account_refresh()
 
 
+def handle_password(type, prompt, email, password, use_keyring=False):
+    if not password:
+        # If we don't yet have a password, try prompting for it
+        password = keyring.get_password(type, email)
```

Comment:
Why do we try to get the password from the keyring when use_keyring is False?

Reply:
That's a great question. Previously, when we supported python V2, we checked if keyring was installed and if a password was supplied or not. Now, keyring is always imported, so we are really just seeing if a password was supplied or not.  However, now looking at the code again, we probably should have changed the conditional when we removed support for v2 as python was always installed. At this point, it'd make sense to only use keyring if the user wants to. In addition, I think it might make sense in v2 to ALWAYS use keyring for password management unless the user submits the password at runtime. 

Thoughts?

Reply:
Makes sense. Just checking for `use_keyring` makes sense for this PR, and I also agree changing to some more modern defaults for v2 sounds good, including keyring, and we can add a --no-keyring option if someone really wants a prompt.